lib/Report/DistributionReport.php

```diff
@@ -257,6 +257,7 @@ public function setReportScheduleFormData()
     /** @inheritdoc */
     public function generateSavedReportName($filterCriteria)
     {
+        $saveAs = '';
```

Comment:
I might be wrong, sorry if I am, but I think we can still get to the end of this method without setting `$saveAs` to be anything other than empty string?

Reply:
yes, we can get to end and no error is thrown.

Reply:
does that mean I can remove the line $saveAs = '';  ?